scm/driver/stash/stash.go

```diff
@@ -79,26 +79,29 @@ func (c *wrapper) do(ctx context.Context, method, path string, in, out interface
 		case *contentCreateUpdate:
 			// add the content to the multipart
 			var b bytes.Buffer
-			w := multipart.NewWriter(&b)
+			mw := &scm.MultipartWriter{Writer: multipart.NewWriter(&b)}
 			// add the other fields
-			if content.Message != "" {
-				_ = w.WriteField("content", string(content.Content))
+			if string(content.Content) != "" {
```

Comment:
the empty string checks are handled within the `mw.Write` method, so could probably lose these additional checks. 

Though see comment below on whether the general purpose struct should be checking for empty values

Reply:
done